client/src/components/tweet/tweet.tsx

```diff
@@ -32,7 +32,7 @@ export default function TweetCard(props: { tweet: Tweet; account: any; profile:
       .filter((m) => m.type === "photo")
       .map((photo, index) => {
         const { media_url_https } = photo;
-        return <ReactIntense key={index} src={mediaDataUrl(media_url_https)} />;
+        return <img key={index} alt="" src={mediaDataUrl(media_url_https)} />;
```

Comment:
This is a pretty big hit to usability when browsing tweets. I'm guessing the component isn't up to date?

Reply:
Thanks for pointing this out again, it seems the latest version of ReactIntense had the issue, rolling back to the older version solves it.

Reply:
Opened an issue on their project
https://github.com/brycedorn/react-intense/issues/30